### PR TITLE
feat: Added support for all storages using s3 protocol

### PR DESCRIPTION
**Pull request by kirandevtn**
Assigned a task to add the support for all storage types using s3 protocol eg. MINIO,AWS etc.
Added the support for the same
Added a variable to get the end-point of the minio or storage.
Also added the the secrets file for each storage provider namely: gcp,aws,azure,s3 and asked the base64 encoded strings.
Also solved the error getting generated while scheduling cronJob at * * * * *
Also resolved the issue for GCP backup not using argocd.

### CODE REVIEW

**Comment by pawan-mehta-dt** [comment]
Update chart version as well @kirandevtn

**Comment by kirandevtn** [comment]
> Update chart version as well @kirandevtn

> Done! @pawan-mehta-dt

File: charts/devtron-backups/templates/backup-on-gcp.yaml
Comment (pawan-59) [inline charts/devtron-backups/templates/backup-on-gcp.yaml:1]: Use single secret resource here.

Comment (pawan-59) [reply charts/devtron-backups/templates/backup-on-gcp.yaml:1]: Resolved

**Review by pawan-59 — APPROVED** [review]

File: charts/devtron-backups/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Devtron Backup Solution
+
+This chart contains configuration files for setting up backup solutions for Devtron stack using different cloud storage providers and persistence options.
```
Comment (pawan-mehta-dt) [inline charts/devtron-backups/README.md:3]: remove solutions from description

File: charts/devtron-backups/values.yaml [hunk 2]
```diff
@@ -33,7 +35,7 @@ global:
       # Only Encoded JSON CREDENTIALS allowed. Example shown below.
```
Comment (pawan-mehta-dt) [inline charts/devtron-backups/values.yaml:35]: Specify base64 encoded instead of only encoded

File: charts/devtron-backups/README.md (hunk 1 above)
Comment (kirandevtn) [reply charts/devtron-backups/README.md]: Did the changes.

File: charts/devtron-backups/values.yaml (hunk 2 above)
Comment (kirandevtn) [reply charts/devtron-backups/values.yaml]: Done!

**Comment by pawan-mehta-dt** [comment]
LGTM

**Pull request merged**